SOD.Common/Custom/InteractableInstanceData.cs

```diff
@@ -10,6 +10,10 @@ public sealed class InteractableInstanceData
     {
         internal InteractableInstanceData(Interactable interactable, InteractablePreset preset)
         {
+            if (interactable == null)
+            {
+                throw new System.NullReferenceException("Interactable is null");
```

Comment:
I feel like this is pretty hacky, the patches instantiate these objects so if through the patch is null you get exceptions?
Doesn't seem very user friendly?